app/views/katello/api/v2/organizations/show.json.rabl

```diff
@@ -2,8 +2,7 @@ object @taxonomy
 
 extends "api/v2/taxonomies/show"
 
-attributes :task_id, :label, :system_info_keys, :distributor_info_keys, :default_info,
-  :owner_details, :redhat_repository_url
+attributes :task_id, :label, :owner_info, :redhat_repository_url
```

Comment:
Intentional to change from `owner_details` to `owner_info` ? If so, you will need to consider how to handle this UI expectation -- https://github.com/Katello/katello/blob/2693b3e84c7f37fa987682aac181a92b946148b2/engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/manifest/manifest-details.controller.js#L34